### ISSUE TITLE
Clock loses time

### ISSUE THREAD

**Issue by stefan73** [chevyhu/Flysky-OpenTX-Test-Branch#18]
Current firmware can set the time. After a day with multiple switch on/off, time is incorrect. Seems reproducible.

**Comment by arhifisyant** [comment]
yes. i also experience the same. my time is slowly drift. Faster than the actual clock. noticeable. I have 16 mins faster. last time I sync the time was about 2 month ago
